### PR TITLE
PROD-6616 - Version 0.0.1

### PR DESCRIPTION
**Pull request by Dimitris-Ilias**
**Version 0.0.1**
Has already been tested locally, some more unit tests missing in order to bump this to version `1.0.0`

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+test/
+.nyc_output
```
Comment (nikostoulas) [inline .npmignore:1]: Add node_modules too here. Check also https://github.com/Workable/rabbit-queue/blob/3b396b590924147aaeefa934f91382f3c83c52a4/.npmignore

File: README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+## Installation
+
+```bash
+$ npm install passport-indeed-oauth2
+```
+
+## Usage
+
+#### Basic Usage
```
Comment (nikostoulas) [inline README.md:16]: 👏 Good job including sufficient Readme instructions

File: package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
+
+{
+    "name": "passport-indeed-oauth2",
+    "version": "1.0.0",
```
Comment (nikostoulas) [inline package.json]: versions >= 1.0.0 are stable. We could make the first one 0.0.1 etc until unit tests are finalized etc.

**Review by nikostoulas — APPROVED** [review]

File: .npmignore (hunk 1 above)
Comment (Dimitris-Ilias) [reply .npmignore:1]: Added

File: package.json (hunk 3 above)
Comment (Dimitris-Ilias) [reply package.json]: Done

**Review by nikostoulas — APPROVED** [review]

**Pull request merged**